ivadomed/models.py

```diff
@@ -325,14 +324,13 @@ def __init__(self, in_channel=1, depth=3, drop_rate=0.4, bn_momentum=0.1, n_meta
         self.down_path.append(max_pool(2))
 
         for i in range(depth - 1):
-            self.down_path.append(DownConv(n_filters, n_filters * 2, drop_rate, bn_momentum, is_2d))
-            self.down_path.append(FiLMlayer(n_metadata, n_filters * 2) if film_layers and film_layers[i + 1] else None)
+            self.down_path.append(DownConv(n_filters*2**i, n_filters*2**(i+1), drop_rate, bn_momentum, is_2d))
```

Comment:
This changes the behaviour. What is the justification? Maybe we should add a unit test to cover these lines of code?

Reply:
@jcohenadad 
I think that it doesn't change the behavior, just the counter (n_filters) replaced by the counter of the loop for fewer lines of code.

Reply:
but what about the `film_layers` condition?

Reply:
about testing the validity of the code, I checked the output of each layer by entering an input tensor to the code and the related Jupyter notebook file can be found in the following link:
https://github.com/AshkanTaghipour/Attention_2D_Unet/blob/main/2D_attention_Unet.ipynb

however, any specific link or related subject about how to add a unit test would be appreciated.

Reply:
here: https://github.com/ivadomed/ivadomed/wiki/tests

Reply:
about film_layers condition, the logic is the same, just similar to the previous version, the second argument of the FiLMlayer should have the size of the second argument of DownConv which it has.

Reply:
The Unit test will be added.